pkg/bootconfig/zipconfig.go

```diff
@@ -6,16 +6,25 @@ package bootconfig
 
 import (
 	"archive/zip"
+	"bytes"
+	"crypto"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha512"
+	"crypto/x509"
+	"encoding/pem"
 	"errors"
 	"fmt"
 	"io"
 	"io/ioutil"
 	"log"
 	"os"
 	"path"
+	"path/filepath"
+	"strings"
+	"time"
 
-	"github.com/u-root/u-root/pkg/crypto"
-	"golang.org/x/crypto/ed25519"
+	urootcrypto "github.com/u-root/u-root/pkg/crypto"
```

Comment:
ucrypto is our convention for u-root packages with the same name as stdlib stuff

Reply:
we don't need this anyway. Removed it in commit ef069fe